lib/order.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'texting'
+
+class Order
+
+  attr_reader :basket
+
+  def initialize
+    @basket = {}
+  end
+
+  def add_to_basket(menu, dish_number, quantity)
+    dish = menu.select(dish_number)
+    quantity.times do
+      if basket.keys.include?(dish)
+        @basket[dish] += dish.price
+      else
+        @basket[dish] = dish.price
+      end
+    end
+  end
+
+  def summary
+    fail "Basket empty" if empty_basket?
+    basket.each_pair do |dish, cost|
+      puts "#{(cost / dish.price).round}x #{dish.name} | £#{format('%.2f', cost)}"
```

Comment:
Metrics/LineLength: Line is too long. [82/80]<br>Rails/Output: Do not write to stdout. Use Rails' logger if you want to log.